deploy/crds/sriovnetwork.openshift.io_sriovnetworknodepolicies_crd.yaml

```diff
@@ -37,12 +37,20 @@ spec:
               description: The driver type for configured VFs. Allowed value "netdevice",
                 "vfio-pci". Defaults to netdevice.
               enum:
-              - netdevice
-              - vfio-pci
+                - netdevice
```

Comment:
Did you use operator-sdk to generate the CRDs? Why adding space here?

Reply:
Yes, it is a space generate by auto format tool I use, will remove it

Reply:
I see it is adding some fields and removing other fields also it is changing the `apiVersion`, can you please cooperate, which changes to keep, attached the diff file when run `make gencode'

Reply:
I get the feeling code generation might change between operator-sdk versions....

@pliurh does this project have any guideline on how to update CRDs after changing the corresponding `_types.go` file

Reply:
That is true. Here are some guideline I can think of:

1. Normally we shall use the `operator-sdk` command with the same version as https://github.com/openshift/sriov-network-operator/blob/97d4e33c781224c0758f15c05619fa78f2c5567b/go.mod#L21
2. the NetworkAttachmentDefinition shall always be ignored, as it was not generated by `operator-sdk`. 
3. `preserveUnknownFields: false` shall be kept. Otherwise, `oc explain` wouldn't work.
4. Don't touch any existing fields if it is not necessary.